### ISSUE TITLE
Copilot Chat icon not visible on latest VS Insiders + Copilot Nightly

### ISSUE THREAD

**Issue by 1000010** [microsoft/vscode-copilot-release#253]
- VS Code Version: 1.80.0-insider
- GitHub Copilot Nightly  v1.89.163 
- OS Version: Windows 11
- Logs:

Steps to Reproduce:

1. Install VS Code (Insiders or stable, tried both) and Copilot Nightly extensions
2. Copilot suggestions work, panel as well. But there is not Copilot chat icon in Activity Bar and chat cannot be accessed
![copilot_bug](https://github.com/microsoft/vscode-copilot-release/assets/6886855/c4af3328-f91c-4ef1-b4ec-c4ce685b86f5)
[exthost.txt](https://github.com/microsoft/vscode-copilot-release/files/11769809/exthost.txt)

**Comment by JoaoFHS** [comment]
I have the same problem. Additionally, I also tried to run Copilot on a VS Code installation running on Windows Subsystem for Linux(WSL) and still have the same problem.

- VS Code Version: 1.79.2
- GitHub Copilot Version: 1.89.156
- OS Version: Windows 11

**Comment by 1000010** [comment]
That was exactly my test case as well. But it doesn't work in either WSL or native.

**Comment by PCDSandwichMan** [comment]
I ran into the same issue as well.

```
VS Code Version:  1.80.0
GitHub Copilot Version: 1.89.165
OS Version: macOS Ventura 13.4 
```

And was resolved when `GitHub Copilot Chat v0.3.2023061502` was installed.
